README.md

```diff
@@ -243,3 +243,4 @@ does install kubernetes in its simplest form.
 [INSTALL.SSL.md]: docs/INSTALL.SSL.md
 [config.md]: docs/config.md
 [HA.md]: docs/ha.md
+
```

Comment:
@pecameron  Did you still want this commit and the change?

Reply:
@rajatchopra the changes to README.md are out of scope. README.md needs a significant revision.